src/components/Page.js

```diff
@@ -24,21 +24,23 @@ const FloatFooterPageComponentWrapper = styled('div')`
 `;
 
 const Page = ({ Head = Header, Foot = Footer, Component, ...props }) => (
-  <Container>
+  < Container >
     <Head />
     <FloatFooterPageContentWrapper>
       <FloatFooterPageComponentWrapper>
         <Component {...props} />
       </FloatFooterPageComponentWrapper>
       <Foot />
     </FloatFooterPageContentWrapper>
-  </Container>
+  </Container >
 );
 
 export const FixedFooterPage = ({ Head = Header, Foot = Footer, Component, ...props }) => (
   <Container height="auto">
     <Head />
-    <Component {...props} />
+    <FloatFooterPageContentWrapper>
+      <Component {...props} />
+    </FloatFooterPageContentWrapper>
```

Comment:
This solution unfortunately would not work out very well as it affects other pages. 
I'd suggest a modification to the cohort builder layout in `src/components/CohortBuilder/index.js` would be a better solution, probably with a `min-height` applied somewhere :) @ciaranschutte is your expert here ;)

![screen shot 2019-02-28 at 10 10 44 am](https://user-images.githubusercontent.com/11821106/53576238-3a632580-3b41-11e9-9458-a6c57f34d1f6.png)

Reply:
Add code changes to style of  **kf-portal-ui/src/components/CohortBuilder/index.js** to fix this bug